content/git/concepts/reset/reset.md

```diff
@@ -0,0 +1,88 @@
+
+## Syntax
+
+In the terminal, the "git reset" command can be used by typing the following:
+
+```pseudo
+git reset <mode-option> <commit-reference>
+```
+
+**Note:** HEAD refers to the current commit while index refers to the staging area where files are [added using Git](https://www.codecademy.com/resources/docs/git/add).
+
+### Mode Options
+
+The `<mode-options>` refer to how far `reset` will go when rolling back changes to a previous commit, including:
+
+- Where the `HEAD` is pointing towards (usually done with just `git reset`).
+- Whether the staging area or Index, reflects the commit the `HEAD` is now pointing towards.
+- Whether the working tree is rolled back to reflect the changes reset in the `HEAD` and staging area.
+
+More specifically, these options include:
+
+- `--soft`: This rolls back to an earlier commit by moving the `HEAD` pointer towards it but leaving the staging area and working tree untouched, allowing for new commits to be made (runs by default).
+- `--mixed`: In addition to moving the `HEAD` pointer to an earlier commit, the staging area is cleared to reflect the changes made in that commit (this option runs by default).
+- `--hard`: This goes one step further and resets the working tree to reflect the previous commit reflected in the staging area and the `HEAD` pointer.
+
+The commit can either be the commit hash generated by Git, or in reference to the HEAD.
+
+## Example
+
+Suppose an error was made in a file named **example.txt** and was accidentally added and committed to a Git repository, `git reset` can be used to go back to the state before that commit was made. The default `--mixed` mode resets the `HEAD` and staging area back to the previous commit.
+
+
+This is what the terminal would look like after creating a new commit on the `main` branch and running [a `git status` check](https://www.codecademy.com/resources/docs/git/status):
+
+```
+On branch main
+Your branch is up to date with 'origin/main'.
+nothing to commit, working tree clean
+```
+
+The text above indicates the following:
+- The branch is up to date, with the `HEAD` pointing the the most recent commit.
+- There is nothing to commit in the staging area.
+- New changes haven't been made yet in the working tree.
+
+To reset the `HEAD` pointer and the staging area back by 1 commit, one of the following commands can be run:
+
+```
+git reset --hard <commit-hash>
+git reset --mixed HEAD~1
+```
+
+To receive the commit hash, run the following command:
+
+```
+git rev-parse HEAD
+```
+
+If `git status` is run once more, the following should appear on the terminal:
+
+```
+On branch main
+Your branch is behind 'origin/main' by 1 commit, and can be fast-forwarded.
+  (use "git pull" to update your local branch)
+Untracked files:
+  (use "git add <file>..." to include in what will be committed)
+        example.txt
+nothing added to commit but untracked files present (use "git add" to track)
```

Comment:
I tried out this example on my local machine: 

- Create an `example.txt` file.
- Ran `git init` to allow for changes to commit-able.
- Made some changes and a few commits in between.

When I run `git status` the first time, I get what's expected with the "Nothing to commit" part. But after I run `git reset --mixed HEAD~1` and then `git status` again, I get this:

```
On branch master
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   example.txt

no changes added to commit (use "git add" and/or "git commit -a")
```

I think this is the correct output to use because the `HEAD` was moved to the correct commit and this text does indeed show the staging area was cleared (the changes were unstaged) by `reset`.

Reply:
Edited the output! Also, changed the code blocks to ```shell.